lib/view_model/exchange/exchange_view_model.dart

```diff
@@ -305,7 +308,19 @@ abstract class ExchangeViewModelBase with Store {
     }
 
     final _enteredAmount = double.tryParse(amount.replaceAll(',', '.')) ?? 0;
-
+    
+     double minLimit = limits.min ?? 0;
+     double maxLimit = limits.max ?? 0;
+
+     if (_enteredAmount < minLimit) {
+       depositAmount = '';
+       return;
+     }
+
+     if (_enteredAmount > maxLimit && isFixedRateMode) {
+       depositAmount = '';
+       return;
+     }
```

Comment:
- indentation
- if the `limits.max` is null, then it should be uncapped, you are resetting it to zero so the condition will always be true.
- also why are you checking on whether its a fixed rate or not with the maxlimit?

Reply:
Fixed the indentation, made it uncapped when the `limits.max` is null and removed the check